File: user-service/src/main/java/rs/raf/user_service/domain/dto/RequestPasswordResetDto.java
package rs.raf.user_service.domain.dto;

import lombok.Data;

import javax.validation.constraints.Email;
import javax.validation.constraints.NotNull;

@Data
public class RequestPasswordResetDto {
    @NotNull(message = "Email cannot be null")
    @Email(message = "Email should be valid")
    private String email;
}

File: user-service-kotlin/src/main/kotlin/pack/userservicekotlin/domain/dto/activity_code/RequestPasswordResetDto.kt
package pack.userservicekotlin.domain.dto.activity_code

import jakarta.validation.constraints.Email
import jakarta.validation.constraints.NotNull

data class RequestPasswordResetDto(
    @NotNull(message = "Email cannot be null")
    @Email(message = "Email should be valid")
    val email: String? = null,
)
